code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -167,7 +167,7 @@
 				icon_state = SUGAR
 			if(CHEFSPECIAL)
 				name = "\improper Chef Excellence's Special Sauce"
-				desc = "A potent sauce distilled from the toxin glands of 1000 Space Carp."
+				desc = "A potent sauce distilled from the toxin glands of 1000 Space Carp with an extra touch of LSD, because why not?"
```

Comment:
The description implies that mindbreaker is one of the ingredients of special sauce. But you cannot craft it yourself probably for balance reasons, and that space carps don't actually have poison glands.